model/network_encoder_attention.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python3
```

Comment:
hi, as you may find, we already implement the feature in our last iteration, please remove it